### DISCUSSION TITLE
Mirage Temasını Kullananlar primeFaces güncellemelerini nasıl yapacaklar?

### DISCUSSION THREAD

**Discussion by musttafayildirim** [primefaces/community#2023] [PrimeFaces Templates] [answered]
PrimeFaces 8 ve mirage temasını kullanıyorum, primefaces sürümünü yüksetme kararı aldık ve bunun için uğraşıyoruz, primefaces 10'a yükselttiğimizde datatable'larda sort olayı çalışmadı, daha üst sürümlerde datatable'larda sort çalışıyor fakat o zamanda komponentlerin ikonları vs görüntülenmiyor, fontawesome ile alakalı olan kısmı hallettim fakat selectOneMenu'lerdeki ikonlar vs görüntülenmiyor, primefaces versiyon güncellemesi için yapılması gerekenler nelerdir paylaşabilir misiniz?

**Comment by melloware** [comment]
English only please.

**Comment by kurtomerfaruk** [comment, marked as answer]
You can follow the necessary steps from the link below to upgrade PrimeFaces version
[https://primefaces.github.io/primefaces/14_0_0/#/../migrationguide/migrationguide](url)

**Reply by musttafayildirim** [reply]
I am using PrimeFaces 8 with the Mirage theme. We have decided to upgrade the PrimeFaces version and are working on it. When we upgraded to PrimeFaces 10, the sorting in DataTables did not work. In higher versions, the sorting in DataTables works, but the icons of the components do not display correctly. I have resolved the issue related to FontAwesome, but the icons in SelectOneMenu appear as squares. Can you share the necessary steps specific to the Mirage theme after updating PrimeFaces?

**Comment by musttafayildirim** [comment]
I am using PrimeFaces 8 with the Mirage theme. We have decided to upgrade the PrimeFaces version and are working on it. When we upgraded to PrimeFaces 10, the sorting in DataTables did not work. In higher versions, the sorting in DataTables works, but the icons of the components do not display correctly. I have resolved the issue related to FontAwesome, but the icons in SelectOneMenu appear as squares. Can you share the necessary steps specific to the Mirage theme after updating PrimeFaces?

**Reply by melloware** [reply]
@musttafayildirim if you send me a DM on the Discord channel I can see if I can help with your Mirage theme.

**Reply by musttafayildirim** [reply]
> @musttafayildirim if you send me a DM on the Discord channel I can see if I can help with your Mirage theme.

Okey, Please tell me discord channel name

**Reply by melloware** [reply]
PF Discord: https://discord.gg/gzKFYnpmCY

**Reply by musttafayildirim** [reply]
Thank you so much @melloware

**Reply by nikita-rogovyi** [reply]
@musttafayildirim could you please share your solution?

**Reply by melloware** [reply]
@nikita-rogovyi are you looking to update Mirage to PF 14.0.0? If so just send me a DM on the PF Discord channel.
